lib/pyld/jsonld.py

```diff
@@ -5830,3 +5810,23 @@ def set(self, active_ctx, local_ctx, result):
 _cache = {
     'activeCtx': ActiveContextCache()
 }
+
+# The default JSON-LD document loader.
+try:
+    _default_document_loader = requests_document_loader()
+except ImportError:
```

Comment:
but I think it would be better to not judge by ImportError from the function call but call `import requests` explicitly and catch exception solely from that import

Reply:
Yeah, perhaps.  Should probably catch ImportError and raise something else more generic.  If anyone wants to refactor to do so, please do.  Can probably get by with code as is for now.